deploy/05-operator.yaml

```diff
@@ -15,7 +15,9 @@ spec:
     spec:
       containers:
       - name: console-operator
-        image: openshift/origin-console-operator:latest
+        # image: openshift/origin-console-operator:latest
+        # image: hogwarts.io/gryffindor/expecto-patronus:v4.0.2
+        image: quay.io/benjaminapetersen/console-operator:v4.0.0
```

Comment:
Was this an accident?

Reply:
Yeah fixed right after I pushed. Sorry, I'll use a different file for my local dev/debugging.